media-video/droidcam/droidcam-1.4.ebuild

```diff
@@ -0,0 +1,106 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit readme.gentoo-r1 desktop linux-mod xdg-utils
+
+DESCRIPTION="Turn your mobile device into a webcam"
+HOMEPAGE="https://www.dev47apps.com/"
+SRC_URI="https://github.com/aramg/droidcam/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="gtk"
+RESTRICT="bindist mirror"
+
+DEPEND="gtk? (
+				dev-cpp/gtkmm:3.0
+				media-video/ffmpeg
+			)
+		!gtk? ( media-video/ffmpeg[-X] )
+		=app-pda/libusbmuxd-1*
+		media-libs/alsa-lib"
+RDEPEND="${DEPEND} dev-util/android-tools"
```

Comment:
Not sure, but I think android-tools is a optional dependency, (I think it is only required for USB connection, via wifi should work without the package) but I haven't tested that yet so I'm not sure.

Reply:
Yep, I thought about it a bit while writing it but it does sound reasonable now.